Serializer/PlainTextDeserializationVisitor.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Smartbox\CoreBundle\Serializer;
+
+/**
+ * This is a really simple visitor that will effectively bypass JMS. Useful for REST APIs that are not truly REST APIs
+ * and return plain text responses, which JMS by default refuses to process because they are not real JSON.
+ */
+class PlainTextDeserializationVisitor extends \JMS\Serializer\JsonDeserializationVisitor
```

Comment:
The `PlainTextDeserializationVisitor` should extend from `GenericDeserializationVisitor` rather than from `JsonDeserializationVisitor` because it doesn't need to be related to that

Reply:
Actually `GenericDeserializationVisitor` is deprecated and removed from JMS 2.0 so I went ahead and extended `AbstractVisitor` which is the one available in later versions. Class is a little bit more complex now but it won't hinder us once we want to update JMS.

Reply:
Godd catch, I didn't notice that GenericDeserializationVisitor was deprecated.